packages/sessions-idls/.gitignore

```diff
@@ -0,0 +1,2 @@
+src/idl
+src/types
```

Comment:
i am not very satisfied with the boundary between generated artifacts and not generated here